c++/measures/g4_iw.cpp

```diff
@@ -52,16 +53,18 @@ namespace cthyb {
 
     // Allocate temporary NFFT two-frequency matrix M
     {
-      int resize_factor = 3; // How much bigger should the large mesh bee???
-      int nfreq         = resize_factor * std::max(n_fermionic, n_bosonic - 1 + n_fermionic);
+      // Determine maximum Matsubara index needed
+      int nfreq = std::max(3 * n_fermionic, n_bosonic + n_fermionic);
       gf_mesh<imfreq> iw_mesh_large{beta, Fermion, nfreq};
+      // In case of fermionic measurement, large mesh only needed for first component
```

Comment:
I do not get any error when running the code, but I do get deviations in the result when comparing to a previous calculation.

I am reading up on the NFFT parameters, I think this is possible to fix by tweaking the `m` oversampling factor.